CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Change Log
 
-## unreleased
```

Comment:
Could you keep this `unreleased` title? That way it's there next time opens a PR and needs to add to it, so it makes it harder to miss.

Reply:
good catch; fixed!